src/composables/useFlawCvssScores.ts

```diff
@@ -5,9 +5,17 @@ import { groupWith, equals } from 'ramda';
 import { deleteFlawCvssScores, putFlawCvssScores, postFlawCvssScores } from '@/services/FlawService';
 import type { ZodFlawType } from '@/types/zodFlaw';
 import { deepCopyFromRaw } from '@/utils/helpers';
+import { IssuerEnum } from '@/generated-client';
 
 const formatScore = (score: any) => score?.toFixed(1);
 
+export const issuerLabels: Record<string, string> = {
```

Comment:
Why are we creating the enum here if it is only used on `CvssSection`?
Wouldn't it make more sense to declare it in that file since it is not used elsewhere?

Reply:
IMO it is well placed, as the composable is specific for CVSS and we keep the component script logic minimized.
No strong preference tho.